apps/els_core/src/els_utils.erl

```diff
@@ -449,3 +450,27 @@ base64_encode_term(Term) ->
 -spec base64_decode_term(binary()) -> any().
 base64_decode_term(Base64) ->
   binary_to_term(base64:decode(Base64)).
+
+-spec levenshtein_distance(binary(), binary()) -> integer().
+levenshtein_distance(S, T) ->
```

Comment:
In general the "Jaro distance" algorithm is a better metric to use for comparing identifiers - it make it much easier to compare giving a 0..1 score for similarity between two strings, rather than the number of edits.

It also gives somewhat higher weight to matching prefixes which often also helps.

Elixir uses [`String.jaro_distance/2`](https://hexdocs.pm/elixir/String.html#jaro_distance/2) for exactly this purpose - we could probably port the implementation.